doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc

```diff
@@ -0,0 +1,17 @@
+[id='interacting-with-processes-tasks-special-filter-proc']
+= Defining special filters for tasks
+
+You can create special filters for tasks in *Manage Tasks* and *Task Inbox* using the *Advanced* filters option. You can create a special filter that will provide domain specific columns to be added to a task list and the task variables are enabled as columns. Once you create and apply the special filter, you can view the task variables of the task as columns using the *Show/hide columns* option.
+
+.Procedure
+. In {CENTRAL}, click *Menu* -> *Manage* -> *Tasks* or  click *Menu* -> *Track* -> *Task Inbox*.
+. On the *Manage Tasks* page or the *Task Inbox* page, click the *Advanced Filters* icon.
+. In the *Advanced Filters* pane, enter the name and description of the filter, and click *Add New*.
+. Select the `name` attribute from the *Select column* drop-down list. The content of the drop-down will change to *name != value1*.
+. Click the drop-down again and choose *equals to* as the logical query.
+. Change the value of the text field to the name of the task.
```

Comment:
s/Change the value of the text field to the name of the task./In the text field, enter the name of the task.

Reply:
Changed as suggested.